packages/auth/ios/RNFBAuth/RNFBAuthModule.m

```diff
@@ -1667,7 +1667,8 @@ - (NSDictionary *)firebaseUserToDict:(FIRUser *)user {
       @"uid" : hint.UID,
       @"factorId" : hint.factorID == nil ? [NSNull null] : [self getJSFactorId:(hint.factorID)],
       @"displayName" : hint.displayName == nil ? [NSNull null] : hint.displayName,
-      @"enrollmentDate" : enrollmentDate,
+      @"enrollmentTime" : enrollmentDate,
+      @"phoneNumber" : hint.phoneNumber
```

Comment:
I guess this works without a cast given that we're looping over `FIRPhoneMultiFactorInfo *hint` - however I'd question if this is wise?

What happens if I have a user that has registered a TOTP factor - does it crash?

(not an iOS developer so hoping for a steer/hand here)